plugins/modules/source_control/gitlab/gitlab_user.py

```diff
@@ -389,11 +430,25 @@ def assignUserToGroup(self, user, group_identifier, access_level):
     def updateUser(self, user, arguments, uncheckable_args):
         changed = False
 
+        arguments = sanitize_arguments(arguments)
+
         for arg_key, arg_value in arguments.items():
             av = arg_value['value']
 
             if av is not None:
-                if getattr(user, arg_key) != av:
+                if arg_key == "provider" or arg_key == "extern_uid":
+
+                    new_identity = {
+                        "provider": arguments['provider']['value'],
+                        "extern_uid": arguments['extern_uid']['value']
```

Comment:
Do I interpret it correctly that either none of these two module options must be provided, or both? If yes, this must be mentioned in the options' documentation, and `AnsibleModule()` needs `required_together=[('provider', 'extern_uid')]`.

Reply:
Yes that is correct